packages/headless/src/features/insight-search/insight-search-actions-thunk-processor.test.ts

```diff
@@ -164,4 +164,107 @@ describe('AsyncInsightSearchThunkProcessor', () => {
     });
     expect(processed.automaticallyCorrected).toBe(true);
   });
+
+  describe('query correction processing', () => {
```

Comment:
there is a previous that we had about the did you mean feature, it also need to be included in this `describe`